pkg/node/node.go

```diff
@@ -259,10 +288,9 @@ func (s *CSINodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 	}
 
 	targetPath := getStagingPath(ll, req.GetStagingTargetPath())
-
-	isFakeAttachNeed := false
+	isFakeAttachNeed, isFakeAttachNeedDR := false, s.isFakeAttachNeededForDR(volumeCR)
```

Comment:
Could you write UT for combined output of mocked isFakeAttachNeededForDR as well as ignoreErrorIfFakeAttach with mocked isPVCNeedFakeAttach. Ideally all possible paths {isFakeAttachNeededForDR, isPVCNeedFakeAttach} = [{false,false}{false,true}{true,false}{true,true}]
Checking whether isFakeAttachNeed and isFakeAttachNeedDR do have expected values